polaris-react/src/components/KeyboardKey/KeyboardKey.scss

```diff
@@ -1,23 +1,32 @@
-$key-base-height: 24px;
+$key-base-height: 28px;
+$medium-side-padding: 9.5px;
```

Comment:
if we're only using this in 1 place, i don't think we need a var :)

Reply:
@rcd00 The component is supposed to be a square, so I think you can keep this variant but also use it to set a fixed width? And then maybe rename this variable to be `$key-base-dimension`?

Reply:
Just to confirm, for both small and default, will the component always have only one character? (ie. we will never have `shift` / `cmd` and show `⇧` / `⌘`  or a single letter instead)? 

Also, currently, single letters default to upper case. Looking at the figma, the default variant uses uppercase but the small variant does not. Is this what we should expect?

Reply:
Yup, we should use single characters instead, therefore always having one character and a square container. Great catch on the uppercase on small variant, should be the case.